### ISSUE TITLE
AI asks for 10 turn joint war, player does not commit after 10 turns and AI prompt.

### ISSUE THREAD

**Issue by SpankMyMetroid** [LoneGazebo/Community-Patch-DLL#4857]
_1. Mod version (i.e Date - 4/23):_
10/10

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
Greece asked me if I wanted to do a joint war against England. I said I would join in 10 turns, he ended up DoWing her after about 5. After 10 turns, I am asked to join and click yes, but I'm not at war with England that turn or the next. I didn't misclick since I checked the diplo modifiers to be sure and there was nothing bad about breaking a promise to war. 

 

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

[no war.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2523065/no.war.zip)

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

[no joint war.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2523067/no.joint.war.zip)

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by crdvis16** [comment]
I also just encountered this bug- I was prompted to war on Germany by Spain, agreed to do so in 10 turns, and after 10 turns I got the "the time is now" prompt.  I agreed to the war but was not at war with Germany after.  I ended up loading a previous turn and DoWing manually to try to avoid breaking my promise in case the bug led to bad relations with Spain.  In case it matters, Spain had a DP with the Maya whom I was currently at war with.
